src/wallet/wallet_test.go

```diff
@@ -670,6 +675,80 @@ func TestLoadWallet(t *testing.T) {
 	}
 }
 
+func TestLoadDupWallets(t *testing.T) {
+	wlts := Wallets{}
+	dupWlt := &Wallet{}
+
+	for i := 0; i < 3; i++ {
+		var entries []Entry
+		var dupEntries []Entry
+
+		if i%2 != 0 {
+			ripeBytes := []byte{123, 187, 20, 57, 72, 12, 0, 255, 4, 32, 142, 69, 17, 123, 166, 99, 42, 7, 1, 212}
```

Comment:
If you need a stable address you can copy one from the explorer as a string and parse it with `cipher.MustDecodeBase58Address()`